cfgov/v1/management/commands/update_data_snapshot_values.py

```diff
@@ -0,0 +1,67 @@
+import ast
+
+from django.core.management.base import BaseCommand
+
+from v1.models.browse_page import BrowsePage
+from v1.tests.wagtail_pages.helpers import publish_changes
+
+
+class Command(BaseCommand):
+    help = 'Monthly updates to data snapshot values'
+
+    def add_arguments(self, parser):
+        """Adds all arguments to be processed."""
+        parser.add_argument(
+            '--markets',
+            nargs='?',
+            help='String representation of a list of markets'
+        )
+
+    def get_data_snapshots(self):
+        """ Gets all data snapshots from browse pages
+        Assumes there is one data snapshot per page
+        """
+        snapshots = []
+        for page in BrowsePage.objects.all():
```

Comment:
@richaagarwal thanks for the detailed reply.

One other thought was to have a custom page type. I love the idea of doing this in the CMS; it's just that it feels broken somehow to have to do a "look through all pages, find blocks of type X" to get to the blocks we want. Is it the case that there'll be one of these blocks on a specific set of pages? Could those pages just be a new page type (that inherits from `BrowsePage`, with this one addition), and then they get updated directly?